.ci.yaml

```diff
@@ -256,7 +256,7 @@ targets:
     presubmit: false
     recipe: pub_autoroller/pub_autoroller
     timeout: 30
-    bringup: true # https://github.com/flutter/flutter/issues/131968
+    # bringup: true # https://github.com/flutter/flutter/issues/131968
```

Comment:
We can just delete, right?

```suggestion
```

Reply:
Yeah fixed. I had not saved the file before uploading.